app/components/ActionLinks/index.js

```diff
@@ -3,44 +3,54 @@ import PropTypes from 'prop-types'
 import { Link } from 'react-router-dom'
 import './index.scss'
 
-export const ExternalActionLink = ({ url, testId, image, children }) => (
-  <a href={url} className="action-link link" data-qa={testId}>
-    <img src={image} alt={`${children} icon`} />
-    {children}
-  </a>
+export const ActionLink = ({ url, testId, image, children }) => (
+  <div className="action-links__link">
+    <a href={url} className="action-link link" data-qa={testId}>
+      <img src={image} alt={`${children} icon`} />
+      {children}
+    </a>
+  </div>
 )
 
-ExternalActionLink.propTypes = {
+ActionLink.propTypes = {
```

Comment:
no idea why this was called `ExternalActionLink` when the file is just ActionLink